docs/developer/database.md

```
@@ -20,6 +20,18 @@
 ### api_tokens
 - id, user_id, name, token_hash, token_prefix, scope, expires_at, revoked_at
 
+### usage_periods
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)